src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -19,11 +19,15 @@
 -->
 ## Current Release
 
+## Version 2.9.0
```

Comment:
@jaredmoo please remove this "Version 2.9.0" header you have added and place additions to the change log under "Current Release"